### ISSUE TITLE
RequireCombinedAssignmentOperator does not detect parentheses

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#527]
The `RequireCombinedAssignmentOperator` sniff fails to detect cases with parentheses:

```php
$foo = ($foo + 1); // should be detect to be $foo += 1;
```

**Comment by Majkl578** [comment]
I don't think this is in scope of this sniff. This should be detected by UselessParentheses sniff.
In fixing mode, UselessParentheses would remove the parenthesis, then this sniff should detect and fix useless assignment.

**Comment by carusogabriel** [comment]
@Majkl578 But what if I require a Sniff for require parentheses for binary expressions, like PHPCS does?
